frontend/serverless.yml

```diff
@@ -0,0 +1,70 @@
+service: crossfeed-frontend
+
+frameworkVersion: '3'
+useDotenv: true
+configValidationMode: error
+
+custom:
+  webpack:
+    webpackConfig: 'webpack.lambda.config.js'
+  customDomain:
+    domainName: ${file(env.yml):${self:provider.stage}.DOMAIN, ''}
+    basePath: ''
+    certificateName: ${file(env.yml):${self:provider.stage}.DOMAIN, ''}
+    stage: ${self:provider.stage}
+    createRoute53Record: false
+
+provider:
+  name: aws
+  region: us-east-1
+  endpointType: PRIVATE
+  runtime: nodejs16.x
```

Comment:
Shouldn't this file have already been present for the existing frontend?

Reply:
It's in lz-infra, but not master.